supertokens_python/supertokens.py

```diff
@@ -368,6 +400,7 @@ async def create_user_id_mapping(  # pylint: disable=no-self-use
         self,
         supertokens_user_id: str,
         external_user_id: str,
+        user_context: Optional[Dict[str, Any]],
```

Comment:
user_context is usually always the last arg, so keep that pattern. This is an internal function, so you can remove the `= None` from the other two args.

Reply:
This applies to all the changes in this file.